release-notes.txt

```diff
@@ -1,3 +1,791 @@
+RIOT-2022.01 - Release Notes
+============================
+RIOT is a multi-threading operating system which enables soft real-time
+capabilities and comes with support for a range of devices that are typically
+found in the Internet of Things: 8-bit and 16-bit microcontrollers as well as
+light-weight 32-bit processors.
+
+RIOT is based on the following design principles: energy-efficiency, soft
+real-time capabilities, small memory footprint, modularity, and uniform API
+access, independent of the underlying hardware (with partial POSIX compliance).
+
+RIOT is developed by an international open-source community which is
+independent of specific vendors (e.g. similarly to the Linux community) and is
+licensed with a non-viral copyleft license (LGPLv2.1), which allows indirect
+business models around the free open-source software platform provided by RIOT.
+
+
+About this release
+==================
+
+The 2022.01 release includes:
+
+## List of notable changes here ##
+
+436 pull requests, composed of 1159 commits, have been merged since the
+last release, and 11 issues have been solved. 43 people contributed with
+code in 88 days. 2475 files have been touched with 44925 (+) insertions and
+14687 deletions (-).
+
+
```

Comment:
I would mention here: all the work regarding `rust`, that almost all `boards` where migrated to kconfig. The fact that with `ztimer64` all code can be migrated to `ztimer` from `xtimer`.

Reply:
Yup. Draft summary in the pipe. Stay tuned (and thanks for the suggestions @fjmolinas )